### PR TITLE
Panel: Transition with new toolbars

### PR DESCRIPTION
**Pull request by cgack**
Fixes gh-8324

### CODE REVIEW

**Comment by cgack** [comment]
@arschmitz this is ready for a round of review

File: js/widgets/panel.js [hunk 1]
```diff
@@ -142,16 +142,20 @@ return $.widget( "mobile.panel", {
 			wrapper = $( "<div>" );
 			this._addClass( wrapper, "ui-panel-wrapper" );
 			wrapper = this._safelyWrap( thePage, wrapper,
-				this._page().children( ".ui-header:not(.ui-header-fixed), " +
-					".ui-content:not(.ui-popup), .ui-footer:not(.ui-footer-fixed)" ) );
+				this._page().children( ".ui-toolbar-header:not(.ui-toolbar-header-fixed), " +
+					"[data-role='toolbar'],"  +
```
Comment (arschmitz) [inline js/widgets/panel.js]: need to add in $.ui.ns here for now to support chaning that until its removed in 1.6 and will be data-ui-toolbar

**Comment by cgack** [comment]
@arschmitz ready for another round of review on this fix

**Comment by arschmitz** [comment]
landed in 1.5-dev

**Pull request closed without merging**